src/x-platform/electron.js

```diff
@@ -17,6 +18,24 @@ if (window.require) {
 
 const notificationClickTimeout = 20000
 
+// http://crocodillon.com/blog/parsing-emoji-unicode-in-javascript
+const emojiRegExp = new RegExp([
```

Comment:
how do we know it is just those 3 emojis, not more ... maybe just convert all of them?